components/aws/sagemaker/tests/integration_tests/resources/config/xgboost-mnist-trainingjob-debugger/config.yaml

```diff
@@ -0,0 +1,59 @@
+PipelineDefinition: resources/definition/training_pipeline.py
+TestName: xgboost-mnist-trainingjob-debugger
+Timeout: 86400
+ExpectedTrainingImage: ((XGBOOST_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/sagemaker-xgboost:1.0-1-cpu-py3
+Arguments:
```

Comment:
good that we have an integration test but on the other hand we are missing integration test for `debugger_component_demo` under aws-samples. Another thing to consider it is sufficient to have one test for this feature. 

So, here is what I would do, change this test to run against the aws-samples one since that is essential for customers and revert changes in `resources/definition/training_pipeline.py`. Since the sample runs only in us-east-1, I would recommend removing it from canary

Reply:
@Dustin
Can you create a new PR for this change ?